### DISCUSSION TITLE
Sort options

### DISCUSSION THREAD

**Discussion by ghost** [jarun/nnn#1702] [Q&A]
[The docs say](https://github.com/jarun/nnn/wiki/Usage#program-options):

> The keys for option -T are apparent disk usage / disk usage / extension / revision / size / time / version. Capitalize to reverse (except r).

1. What is "revision" and "version"?
2. What is "time"? Is it access time, modified time, creation time?
3. How can I sort alphabetically by file name?

**Comment by N-R-K** [comment]
1. [Version sort](https://www.gnu.org/software/coreutils/manual/html_node/Version-sort-overview.html) is the same as GNU's `sort -V`.

2. You can select which time to use by using <kbd>T</kbd>. I believe it should be modification time by default.

3. Version sort likely does what you want.

**Comment by ghost** [comment]
Ok, thanks.

We still miss the difference between "version" and "revision".

I think the wiki should better document this.

**Reply by ghost** [reply]
> We still miss the difference between "version" and "revision".

It's sad when good software is plagued by lack of proper documentation
